docusaurus/docs/core-concepts/security/authorization/policies/_create-edge-router-policy-cli.md

```diff
@@ -1,4 +1,4 @@
-To create an Edge Router Policy using the CLI issue the following commands. (ensure you are [logged in](/docs/manage/cli/logging-in))
+To create an Edge Router Policy using the CLI issue the following commands. (ensure you are [logged in](/docs/manage/04-cli/logging-in.md))

```

Comment:
with docusaurus, I've found that it's a bitt better in the long run to not use the .md in the link. if the .md is renamed to .mdx it won't break all the links to it.

Reply:
Yes, I noticed, but I thought I got an error once, so was bit confused. It makes sense though if oyu can do that. I guess that relates to the number in front of it as well. For example, 04-cli/05-logging.md, can be as cli/logging

Reply:
If I remove the file type in the link, I get error `broken links` when I run ```yarn build```

Reply:
```
[ERROR] Unable to build website for locale en.
[ERROR] Error: Docusaurus found broken links!

Please check the pages of your site in the list below, and make sure you don't reference any path that does not exist.
Note: it's possible to ignore broken links with the 'onBrokenLinks' Docusaurus configuration, and let the build pass.

Exhaustive list of all broken links found:

- On source page path = /docs/core-concepts/security/authorization/policies/creating-edge-router-policies:
   -> linking to /docs/manage/04-cli/02-logging-in

- On source page path = /docs/core-concepts/security/authorization/policies/creating-service-edge-router-policies:
   -> linking to /docs/manage/04-cli/02-logging-in

- On source page path = /docs/core-concepts/security/authorization/policies/creating-service-policies:
   -> linking to /docs/manage/04-cli/02-logging-in

- On source page path = /docs/core-concepts/security/authorization/policies/overview:
   -> linking to /docs/manage/04-cli/02-logging-in
```

Reply:
The correct path is `/docs/manage/cli/cli-login` because you have overridden the category for the `cli` branch of the folder tree.

Reply:
It's a tradeoff. You could use your original approach of the relative path to the MD(X) file and let Docusaurus resolve it at build time. At least this way it's guaranteed to resolve to a valid link. If we assume the structure of the category is unlikely ever to change then the absolute path to the current, rendered location of the article will be valid even if the filename suffix changes. Personally, I prefer to link to the relative filename and let Docusaurus figure out the HTTP URL path.

Reply:
ok, I see the value of IDs now more clearly. Updated the links to use them instead of file names.

Reply:
If you are ok with this, I will close this issue. 
```
[logged in](/docs/manage/cli/cli-login))
[update section](update/router-update/#update-router)
etc
```

Reply:
That's a valid reference for `logged in`. 

For `update section`, you'll have to make a similar choice between the **absolute URL path** (starts with `/`) and **relative URL path** (starts with `./`, preferably) , and **relative file path** (starts with `./`, preferably, and ends with a filename suffix). The current value doesn't look valid. Does it build like that?

Reply:
the relative path works both ways, i.e ./router or just router. Not sure if it makes any difference which one to use.

Reply:
I use `./` because it's explicit and triggers my IDE to auto-complete the reference.

Reply:
ok, since most people use that I assume. Then , it is ```./``` :)